includes/discovery/processors/avaya-ers.inc.php

```diff
@@ -1,15 +1,9 @@
 <?php
 
-if ($device['os'] == 'avaya-ers') {
-    // Processor information only know to work with 3500, 5500 and 5600 switches
-    if (preg_match('/[35][56][0-9][0-9]/', $device['sysDescr'])) {
-        $procs = snmp_walk($device, '.1.3.6.1.4.1.45.1.6.3.8.1.1.6', '-Osqn');
```

Comment:
You've removed this from your update so it won't run :)

And you also need the `if ($device['os'] == 'avaya-ers') {` still.